builds.yml

```diff
@@ -59,7 +59,7 @@ buildTypes:
       - build-flask
       - keyring-snaps
       - blockaid
-      - petnames
+      - simulation-preview
```

Comment:
We don't really want to add simulations to any build yet as it's incomplete and we're using this feature as a means of prevent any partial release.

I'm now realising the downside of this is our E2E tests can't run in the CI so perhaps we do local runs or test in the CI, but then include `it.skip` until the code fence is removed?